ETTrace/Symbolicator/Symbolicator.swift

```diff
@@ -153,11 +153,7 @@ public class StackSymbolicator {
         let strs = addrsArray.map { String($0 + addition, radix: 16) }
         try! strs.joined(separator: "\n").write(toFile: addrsFile, atomically: true, encoding: .utf8)
 
-        let arch = try? safeShellWithOutput("/usr/bin/file \"\(binary)\"").contains("arm64e") ? "arm64e" : "arm64"
```

Comment:
The arch flag doesn't seem to be needed as long as it isn't a universal binary, and this wasn't making an effort to pick the right arch from a universal binary anyways